ui/app/AppLayouts/Chat/components/StickerPackPurchaseModal.qml

```diff
@@ -7,7 +7,7 @@ import "../../../../shared"
 
 ModalPopup {
     id: root
-    property var asset: { "name": "Status", "symbol": walletModel.getStatusTokenSymbol() }
+    readonly property var asset: JSON.parse(walletModel.getStatusToken())
```

Comment:
I think it's preferable to put that in a `try/catch`, otherwise the whole component stops to work in case of a bad parse

Reply:
Very true, except that the value from the function `getStatusToken()` is essentially hardcoded so I don't think it's necessary